src/Relation/Binary/Bundles.agda

```diff
@@ -107,7 +110,7 @@ record TotalPreorder c ℓ₁ ℓ₂ : Set (suc (c ⊔ ℓ₁ ⊔ ℓ₂)) where
   preorder = record { isPreorder = isPreorder }
 
   open Preorder preorder public
-    using (module Eq)
+    using (module Eq) renaming (_≁_ to _⋦_)
```

Comment:
I think it may also be necessary to hide `_∼_` here, so that it is not  re-exported?

Reply:
I don't think you ever need both `using` and `hiding`?

Reply:
Yes, you're right. I was scratching my head trying think how to suppress spurious manifest fields in a `record`. Maybe it's all ok after all, but I do worry that in `Preorder`screamed as substructures if some other ordering type, the fields if that record are going to have the wrong names wrt the ambient enclosing ordering. But no obvious bugs so far...???